src/Console/Commands/OptimizeCommand.php

```diff
@@ -43,6 +43,9 @@ public function handle(
         SearchableFinder $searchableFinder,
         LocalSettingsRepository $localRepository
     ) {
+        if ($prefix = $this->option('prefix')) {
```

Comment:
Please run `./vendor/bin/phpstan` to make sure this is working as expected.

Reply:
`[OK] No errors`, so no issue here.